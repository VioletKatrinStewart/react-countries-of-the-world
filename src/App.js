import { useState, useEffect } from 'react';
import './App.css';
import { getCountries } from './services/Countries';

function App() {
  const [Countries, setCountries] = useState([]);
  useEffect(() => {
    const fetchData = async () => {
      const data = await getCountries();
      console.log(data);
    };
    fetchData();
  }, []);
  return <div className="App"></div>;
}

export default App;
