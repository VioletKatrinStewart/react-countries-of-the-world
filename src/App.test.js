import { render } from '@testing-library/react';
import App from './App';

test.skip('renders header ', () => {
  const app = render(<App />);
  expect(app).toMatchSnapshot();
});
