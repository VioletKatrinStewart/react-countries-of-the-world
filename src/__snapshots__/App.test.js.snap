// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <input
          class="input"
          placeholder="search"
          type="text"
          value=""
        />
        <select>
          <option
            value="All"
          >
            All
          </option>
          <option
            value="Africa"
          >
            Africa
          </option>
          <option
            value="Antarctica"
          >
            Antarctica
          </option>
          <option
            value="Asia"
          >
            Asia
          </option>
          <option
            value="Oceania"
          >
            Oceania
          </option>
          <option
            value="Europe"
          >
            Europe
          </option>
          <option
            value="North America"
          >
            Norh America
          </option>
          <option
            value="South America"
          >
            South America
          </option>
        </select>
        <header>
          Countries of the World
        </header>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <input
        class="input"
        placeholder="search"
        type="text"
        value=""
      />
      <select>
        <option
          value="All"
        >
          All
        </option>
        <option
          value="Africa"
        >
          Africa
        </option>
        <option
          value="Antarctica"
        >
          Antarctica
        </option>
        <option
          value="Asia"
        >
          Asia
        </option>
        <option
          value="Oceania"
        >
          Oceania
        </option>
        <option
          value="Europe"
        >
          Europe
        </option>
        <option
          value="North America"
        >
          Norh America
        </option>
        <option
          value="South America"
        >
          South America
        </option>
      </select>
      <header>
        Countries of the World
      </header>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
